<launch>
    <arg name="veh"  doc="Name of vehicle. ex:  "/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />
    <!-- <arg name="testing" default="false" doc="using for test purposes." /> -->

    <arg name="pkg_name" value="sensor_fusion"/>
    <arg name="node_name" default="sensor_fusion_node"/>

    <group ns="$(arg veh)">
        <node  name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" >
            <rosparam command="load" file="$(find sensor_fusion)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
        </node>
    </group>

    <!-- Publications -->

    <!-- Subscriptions -->


</launch>